docs/3.x/angularjs-getting-started.md

```diff
@@ -39,11 +44,11 @@ I recommend watching some videos and read a few resources about the benefits and
   * Some directives such as ng-if and ng-repeat create their own scope
   * All bound data should be dotted when using ng-model otherwise you are doing it wrong. You would be wondering why a value is not updated and it will take you hours to figure this out.
     Example:
-    ```<div ng-model="model.searchfield">``` instead of ```<div ng-model="searchfield">```
+    `&lt;div ng-model="model.searchfield">` instead of `&lt;div ng-model="searchfield">`
```

Comment:
Maybe this won't be displayed properly in markdown ie. will be displaye as html entities?

Reply:
Nope, it was before shown wrong, then I fixed to the entities to make sure it is shown properly.

Reply:
Funny, I had to change it after all. Locally it works like this because I remember that I needed to change it to make it work. I must be using an outdated composer snapshot and an older version of the markdown lib or so.